CHANGELOG.md

```diff
@@ -1,18 +1,29 @@
 # Change Log
 
+## v2.0.1 (2020/10/21)
+
+Patch Update
+
+- Added Abbreviations for CC Licenses
+- Added workaround for webp images on Safari (iOS + MacOS)
```

Comment:
webp?

Reply:
Yeah, so that's the image format optimized for web. It came out in 2010, but Apple still doesn't support it. So added a workaround such that the browser picks the images it supports either jpeg/png/webp etc